src/core/state_data.h

```diff
@@ -13,29 +31,39 @@ class RobotState {
     theta += d_theta;
   }
 public:
-  double x, y, theta;
+  double x, y, theta; ///< Position of robot
 };
 
 
 #include "maps/grid_map.h"
 
 // TODO: try to simplify template params
+/**
+ * Class that knows everything about robot state, can update its position and handle observation
```

Comment:
Change to smth like that "The holder of robot's merged perceptions of an environment"